ci/logging_tests_controller.yml

```diff
@@ -0,0 +1,121 @@
+---
+- name: "Verify logging projects, endpoints, credentials, nodes, pods, services, manifests and subscriptions"
+  hosts: controller
+  gather_facts: no
+  ignore_errors: true
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  vars:
+    common_pod_test_id: "RHOS0-12672"
```

Comment:
I suggest to have a loop here with list in a vars file to centralizes configuration and avoids the duplication.
for example : 

common_pod_tests:
  - test_id: "RHOSO-12672"
    namespace: "openstack-operators"
    status: "Running"
    pod_list:
      - telemetry-operator-controller-manager
      - dataplane-operator-controller-manager
  - test_id: "RHOSO-12752"
    namespace: "openstack"
    status: "Running"
    pod_list:
      - openstackclient
      
```
    - name: "Run pod verification tests"
      ansible.builtin.include_tasks: common_verify.yml
      loop: "{{ common_pod_tests }}"
      loop_control:
        loop_var: test_case

```

Reply:
I will add this as a follow up task for now .